nservicebus/operations/async-analyzer.md

```diff
@@ -19,9 +19,9 @@ The analyzer will point out missing `await` operators when using asynchronous me
 
 ## Limitations
 
-This analyzer inspects only the APIs contained on the classes and interfaces as described in the previous section. Other asynchronous operations on NServiceBus APIs are not analyzed.
+This analyzer inspects only the APIs contained on the classes and interfaces as described in the previous section. Other asynchronous methods on NServiceBus APIs are not analyzed.
```

Comment:
Should the second sentence be "Other asynchronous NServiceBus APIs are not analyzed." since the first sentence just says "APIs" and nothing about methods, implying that the methods ARE the APIs?